### DISCUSSION TITLE
Support for Firefox

### DISCUSSION THREAD

**Discussion by flcdrg** [mjisaak/skilling-champion-extension#3] [Ideas]
Not sure how different it would be, but as a Firefox user, it would be great to be able to use this extension there too :-)

**Comment by mjisaak** [comment]
Hi David, 

I initially decided not to develop it for firefox mainly because nowadays the firefox usage is very low (~4 %). However, It shouldn't be that much effort since the context menu API is very similar (in fact, the API is modeled on Chrome's "contextMenus" API). So maybe I will add support for it in the future, but I can't promise it.

**Comment by flcdrg** [comment]
No worries. Wow, you've given me a bit of a reality check. I didn't realise Firefox usage had dropped that low. I'm feeling very much in the minority!

**Comment by mjisaak** [comment]
The extension is also [available in the Firefox marketplace](https://addons.mozilla.org/en-US/firefox/addon/mvp-docs-learn-champion/).
